.github/workflows/security.yml

```
@@ -26,69 +29,14 @@ concurrency:
 
 jobs:
   # CodeQL analysis
+  # CodeQL temporarily disabled for release/v0.61.3 (default Code Scanning is enabled and rejects custom uploads).
+  # Re-enable after consolidating to a single CodeQL configuration.
   codeql:
-    name: 🔍 CodeQL Analysis
+    name: 🔍 CodeQL Analysis (disabled)
+    if: false
```

actionlint (1.7.9)

condition "false" is always evaluated to false. remove the if: section

(if-cond)